code/data.py

```diff
@@ -0,0 +1,80 @@
+'''
+Provide processed data ready for consumption by tensorflow model
+Return data always as numpy array
+'''
+
+import numpy as np
+import os
+from collections import Counter
+
+class NlpData:
+
+    data = None
+    data_only_ids = None
+    vocab = None
+    current_batch = 0
+
+    def get_next_batch(self, batch_size):
```

Comment:
Iterators are nice, something that iterates over directory over the `tf.placeholders` inputs is a bonus. From `stanford_code/q3_RNNLM.py`:
```
       for step, (x, y) in enumerate(
            ptb_iterator(data, config.batch_size, config.num_steps)):
            # We need to pass in the initial state and retrieve the final state to give
            # the RNN proper history
            feed = {self.input_placeholder: x,
                            self.labels_placeholder: y,
                            self.initial_state: state,
                            self.dropout_placeholder: dp}
```